spec/models/boarding_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Boarding do
+  let(:boarding) { FactoryBot.build(:boarding) }
+  describe 'when creating a boarding' do
+    it 'has a valid factory' do
+      expect(boarding).to be_valid
+    end
+    it { should validate_presence_of(:time) }
+    it { should belong_to(:trip) }
+    it { should belong_to(:boarding_point) }
```

Comment:
I guess the validation of the time can be checked here too!
Can we add spec for checking `has_one` association with `ticket` as well, then maybe the `nullify `concept can be checked too?
What do you think?